frontend/src/components/dialogs/SecretDialog.vue

```diff
@@ -221,14 +214,8 @@ export default {
       return this.shootsByInfrastructureSecret.length
     },
     shootsByInfrastructureSecret () {
-      const secretName = get(this.secret, 'metadata.name')
-      if (secretName) {
-        const predicate = item => {
-          return get(item, 'spec.secretBindingName') === secretName
-        }
-        return filter(this.shootList, predicate)
-      }
-      return []
+      const name = get(this.secret, 'metadata.name')
```

Comment:
I know the following is not part of this PR but shouldn't we rename `InfrastructureSecret` to `InfrastructureSecretBinding` or `InfrastructureBinding` or `InfrastructureCredentials` to clearly distinguish between secret and the thing that keeps the ref to the secret?